packages/react-tc-ui-kit/src/components/Input/Input.js

```diff
@@ -48,16 +52,26 @@ export default function Input({
           ref={inputRef}
           id={id}
           placeholder={placeholder}
-          className={`tc-input__control ${inputPasswordClassname}`}
+          className={`tc-input__control ${inputButtonClassname}`}
           type={isPasswordVisible ? 'text' : type}
           value={value}
           onChange={onChange}
           {...props}
         />
 
+        {type !== 'password' && actionButton && (
```

Comment:
we can remove this repeating code
i think we should declare new object `currentActionButton` that will be password action or `actionButton` from props
and then we will render it here